content/docs/introduction/comparison.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Comparison to alternatives
+sort_rank: 30
+---
+
+# Comparisons
+
+We are often asked how Xmidt+Webpa compares to some alternative.  It's a great
+question.  By itself Xmidt is difficult to compare directly to other systems
+primarily due to the fundamental differences in architecture & different project
+goals.
+
+Xmidt by design is **not** coupled with any data models.
+This allows for inclusion of all manor of data models.  TR-181, TR-104, custom
+Msgpack or JSON are all no problem.  Any system tightly coupled with one of the
+`TR-181` | `TR-104` | `TR-357` | etc, fundamentally couple the service life of
+the entire ecosystem to the use of these particular models.  Xmidt's architecture
+is designed to avoid this coupling.  None of the other existing services have
+this design goal as a core system value.
+
+Xmidt, Webpa, Codex and the ecosystem are designed to be an 'à la carte'
+solution.  This includes the cloud components, the client components and the
+data models available.  You get the choice of which services you want to deploy.
+The service you need may be to access the existing TR-181 or TR-104 data
+model on your device (Webpa+Webpa Client).  It may be an alternative cloud
+managed event scheduler (Aker Client) or something that is custom and
+proprietary to your needs.
+
+The Xmidt community's focus is to give you the easy to use building blocks so
+you can best assemble the system that works best for your use cases.
+
+# Comparison Table of Xmidt+Webpa to TR-069+ACS & USP
+
+To help provide a basic comparison between the 3 solutions we usually get
+questions about, we are limited the scope to just TR-181 or similar access.
+
+TR-069 and USP/TR-369 appear only available as proprietary only solution end
```

Comment:
```suggestion
TR-069 and USP/TR-369 appear only available as proprietary solutions end
```

Are there multiple solutions or just one? I made it plural because I assume there is more than one.